app/services/articles/feeds/large_forem_experimental.rb

```diff
@@ -211,12 +159,12 @@ def user_following_users_ids
         @user_following_users_ids ||= (@user&.cached_following_users_ids || [])
       end
 
-      def first_half(array)
-        array[0...(array.length / 2)]
+      def first_quarter(array)
```

Comment:
This one change takes the top _quarter_ of results instead of _half_ and presents them in a shuffled order. It's a slight modification that should complement the one we selected for the better.